### PR TITLE
refactor(gitlab): separate package installation

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

**Review by lionelB — COMMENTED** [review]
I'm not a big fan of splitted gitlab-ci files in each subpackages as it made hard to navigate between files and hard to represent the whole process

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -2,30 +2,97 @@
...
+  stage: "Registration"
+  dependencies: []
+
+#
+
+Create namespace:
+  extends: .base_create_namespace_stage
+  stage: "Registration"
+  except:
+    - /^renovate/*
```
Comment (lionelB) [inline .gitlab-ci.yml]: this exclude renovate branch / Pr to trigger namespace creation ?

Comment (douglasduteil) [reply .gitlab-ci.yml]: I will remove this

**Comment by douglasduteil** [comment]
> from @lionelB  
> I'm not a big fan of splitted gitlab-ci files in each subpackages as it made hard to navigate between files and hard to represent the who process

I did it so each package handles it's own process in it's local folder. It's a similar point of view as splitted package.json, Dockerfiles, etc...
If we remove / move a package tomorrow we do not need to think about what to change/remove in over external files that the root `.gitlab-ci.yml` file.

I would like to remove the `.gitlab-ci` folder as end result

**Comment by douglasduteil** [comment]
Outdated

**Pull request closed without merging**